binders/kafka-binder/spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/function/FunctionUnitDeriver.java

```diff
@@ -0,0 +1,15 @@
+package org.springframework.cloud.stream.binder.kafka.streams.function;
+
+import org.springframework.util.StringUtils;
+
+import java.util.Arrays;
+
+public class FunctionUnitDeriver {
+	static String[] getFunctionUnits(String definition) {
+		if(!StringUtils.hasText(definition)) {
+			return new String[]{};
+		}
+		final String[] rawSplitDefinition = definition.split(";");
+		return Arrays.stream(rawSplitDefinition).map(String::trim).toArray(String[]::new);
+	}
+}
```

Comment:
We probably don't need a new class since this functionality is trivial. How about adding this as a new static utility method in `KafkaStreamsBinderUtils`?

Reply:
KafkaStreamsBinderUtils and its methods are all package protected, so breaking the encapsulation might be a problem - Unless we're okay with making it public.

(I still don't get why there isn't a "package and subpackage" visibility in Java, it would make things so much nicer 😭  )

Reply:
We actually made `KafkaStreamsBinderUtils` a public utility class for some other reasons.

Reply:
Thanks @sobychacko , my remote was based on the comment before this one - I've rebased now